src/languages/swift.js

```diff
@@ -77,6 +86,11 @@ export default function(hljs) {
   ];
 
   // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID418
+  const OPERATOR_GUARD = {
+    // Prevent -> from being highlighting as an operator.
```

Comment:
So the default operator stuff is a bit "wide". :-)

Reply:
Yes. `->` is regarded as syntax/punctuation. The Swift grammar also states that this token is reserved and cannot be declared as an operator. However, the characters themselves are valid operator characters, so we need a rule to prevent it from being highlighted as an operator.